src/etc/htmldocck.py

```diff
@@ -125,12 +125,6 @@
 VOID_ELEMENTS = set(['area', 'base', 'br', 'col', 'embed', 'hr', 'img', 'input', 'keygen',
                      'link', 'menuitem', 'meta', 'param', 'source', 'track', 'wbr'])
 
-# Python 2 -> 3 compatibility
```

Comment:
I don't think we should be removing support for Python 2 yet. The scripts already support both versions so it doesn't cost us anything to continue to support both.

Reply:
I don't agree with this: the python 2 support ends this year. We don't want to support an obsolete technology.